feature-libs/storefinder/styles/components/_store-finder-list.scss

```diff
@@ -23,6 +23,13 @@
     .cx-map-col {
       height: 70vh;
     }
+
+    .nav-item {
```

Comment:
we should use a `cx-` class to add the styling here

Reply:
done